include/no_os_spi.h

```diff
@@ -187,7 +208,11 @@ struct no_os_spi_desc {
 	enum no_os_spi_mode	mode;
 	/** SPI bit order */
 	enum no_os_spi_bit_order	bit_order;
+	/** SPI Lanes */
+	enum no_os_spi_lanes   lanes;
+	/** SPI bus platform ops */
```

Comment:
these documentation strings should go in a separate commit, something like "document all no_os_spi structure members"